docs/customization/menu.rst

```diff
@@ -41,32 +40,102 @@ In the example below we are adding a one new item to the Admin panel menu and a
         {
             $menu = $event->getMenu();
 
-            $menu->addChild('backend_main')
-                ->setLabel('Test Backend Main');
+            $newSubmenu = $menu->addChild('new')
+                ->setLabel('New Admin Panel Menu');
+
+            $newSubmenu->addChild('new-subitem')
+                ->setLabel('New Item');
         }
+    }
+
+**2.** After creating your class with a proper method for the menu customizations you need, subscribe your
+listener to the ``sylius.menu.admin.main`` event in the ``app/config/services.yml``.
+
+.. code-block:: yaml
+
+    # app/config/services.yml
+    services:
+        app.listener.admin.menu_builder:
+            class: AppBundle\EventListener\Menu\Admin\MenuBuilderListener\MenuBuilderListener
```

Comment:
I'll go with `AdminMenuBuilder` instead of `AdminMenuListener` as it does not listen for any event.